### ISSUE TITLE
White lines between tiles on mobile Firefox

### ISSUE THREAD

**Issue by robinmetral** [mariusandra/pigeon-maps#123]
I'm seeing white lines between tiles on mobile Firefox (Android). Haven't reproduced on desktop or mobile Chrome.

| Test app | Official demo |
| --- | --- |
| ![photo5865978282208507312](https://user-images.githubusercontent.com/35560568/108222030-b133c780-7138-11eb-9447-808898349e18.jpg) | ![photo5865978282208507313](https://user-images.githubusercontent.com/35560568/108222075-bdb82000-7138-11eb-903a-638749175c45.jpg) |

It happens with every tile provider I've tried, so I think the issue is somewhere in this library. As you can see, it's a big problem for dark maps tiles, in lighter variants the lines are not too obvious.

This might be related to #71. However, I've tried to apply the fix suggested by @zdila in https://github.com/mariusandra/pigeon-maps/issues/71#issuecomment-562038771 (round the transform() values), and it didn't fix it for me.

Does anyone know what the issue is? Happy to open a PR if I understand what the issue is. I'd love to avoid having to use Leaflet or Mapbox just because of this :stuck_out_tongue:

**Comment by zdila** [comment]
It is probably a bug of all Chrome-based browsers. See also https://github.com/Leaflet/Leaflet/issues/6101 or https://github.com/Leaflet/Leaflet/issues/3575.

**Comment by mariusandra** [comment]
I couldn't replicate this, not even on Firefox mobile, so it must be some kind of zoom issue? People [have seen](https://github.com/Leaflet/Leaflet/issues/6101#issuecomment-386802688) this even in Google Maps, so it's a pretty big problem, if it happens.

The [suggested fix](https://github.com/Leaflet/Leaflet/issues/3575#issuecomment-688644225) adapted to pigeon-maps v0.17.2 is something like this:

```css
.pigeon-tiles img {
    width: 256.5px !important;
    height: 256.5px !important;
}
```

I'm very hesitant in including that in the core for obvious reasons though.

**Comment by zdila** [comment]
@mariusandra just for sure - do you have HiDPI screen? On regular DPI screens it is not reproducible.

**Comment by mariusandra** [comment]
Yeah, all my screens are retina. I'm now on the latest M1 Air, use a MBP 16" for work, have a 4K monitor and a 1440p Android phone. No issue on any.

**Comment by zdila** [comment]
In Chrome (95), directly from https://pigeon-maps.js.org/. It is not always visible there, just if you pan a lot then sometimes it appears:
![image](https://user-images.githubusercontent.com/636095/141089682-854181f1-3894-4fb0-9e97-a0e3373c212b.png)

**Comment by jjzazuet** [comment]
I can also confirm that applying the CSS styling fix also corrects rendering on Firefox for MacOS using `preact`. In my case, the tiles appear wider than what they should be (height is correct though).

Before:
<img width="899" alt="Screenshot 2024-09-15 at 3 03 59 AM" src="https://github.com/user-attachments/assets/4a05e16a-a541-4b1a-b08e-6db9a7508e7c">

After:
<img width="899" alt="Screenshot 2024-09-15 at 3 04 32 AM" src="https://github.com/user-attachments/assets/bf0ff7b2-db8f-4ebb-9139-fd3137ab0cd6">
